src/test/java/CalculatorTest.java

```diff
@@ -0,0 +1,72 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+
+    Calculator calculator;
```

Comment:
```suggestion
    private Calculator calculator;
```
접근제어자를 막아주어야하지 않을까요? default를 사용하지 않는 이유와 private을 사용하는 이유가 무엇일까요?

Reply:
default 접근제어자는 같은 패키지에서 접근할 수 있습니다. 따라서 Calculator 단위 테스트에서만 사용되는 필드가 관련없는 클래스에서 오용할 수 있는 가능성이 있네요! 

private은 해당 클래스에서만 접근하게 하여 의도치 않은 실수를 미연에 방지할 수 있을것 같습니다!

수정하도록 하겠습니다!

Reply:
가능하면 앞으로의 미션 내에서도 public, private만을 사용하도록 설계하고 개발해보시는걸 권장드려요. default와 protected는 말씀하신대로 같은 패키지 내에서 혹은 상속된 클래스 내에서 접근이 가능하기 때문에 캡슐화가 깨질 수 있는 반쪽짜리 접근제어자입니다.